backend/api/Services/MissionSchedulingService.cs

```diff
@@ -55,10 +55,17 @@ public async Task StartNextMissionRunIfSystemIsAvailable(string robotId)
                 logger.LogInformation("The robot was ready to start mission, but no mission is scheduled");
                 if (robot.MissionQueueFrozen) { return; }
 
+                if (!await localizationService.RobotIsLocalized(robotId))
```

Comment:
Do we have this covered in a test or should we write one to see what happens if the robot is not localized and robot available is triggered?

Reply:
Test has been added